### ISSUE TITLE
"autoprefixer_browsers" on config.json is not being respected when uploading to BigCommerce

### ISSUE THREAD

**Issue by hectorfhurtado** [bigcommerce/cornerstone#975]
Hello, on _config.json_ file, I'm using this definition for some _-webkit-_ prefixes needed on Safari. It works on development but when uploading into BigCommerce it's like being ignored (they are on the bundle config.json file, so it's being bundled).
```
  "autoprefixer_browsers": [
    "ios 8",
    "ios 9",
    "ios 10",
    "last 4 versions",
    "> 5% in US"
  ],
```
I attached a screen shot for when on development (first one) and on the storefront (second one).

![development](https://cloud.githubusercontent.com/assets/2037113/24118561/3c0e6fce-0d7c-11e7-9036-756040ba72d5.png)
![storefront](https://cloud.githubusercontent.com/assets/2037113/24118568/3eaff8ce-0d7c-11e7-881e-4084b6dbb55e.png)

**Comment by bc-AlyssNoland** [comment]
@hectorfhurtado we are investigating this as STENCIL-3317

**Comment by mjschock** [comment]
there's a PR to fix this now:  https://github.com/bigcommerce/stapler/pull/465

**Comment by mjschock** [comment]
@hectorfhurtado the fix for this issue was included in the 1.7.0 release today:  https://stencil.bigcommerce.com/docs/cornerstone-17x-theme-release-notes.  i'm going to close the ticket, but please reopen it's still not working for you.  anyway, thanks for submitting the issue!

cc @bc-AlyssNoland

**Comment by hectorfhurtado** [comment]
@mjschock thank you! I'll test then
